tests/test_app.py

```diff
@@ -19,6 +19,11 @@ def test_run_json_mode(self):
         # The test data contains fatal qc errors
         self.assertEqual(app.run(), 1)
 
+    def test_run_downgrade_error(self):
+        app = App(runfolder=self.RUNFOLDER, downgrade_error="ReadsPerSampleHandler")
+        # Test data should not produce fatal qc errors anymore
+        self.assertEqual(app.run(),0)
```

Comment:
Nit picking, but there is a missing white-space here in, i.e. it should be: `self.assertEqual(app.run(), 0)`

Reply:
Nit picking is good, fixed!